SPECS-SIGNED/srp-signed/srp-signed.spec

```diff
@@ -99,6 +99,9 @@ popd
 %license %{_datadir}/licenses/%{name}/copyright
 
 %changelog
+* Wed Feb 19 2025 Chris Co <chrco@microsoft.com>
```

Comment:
Missing version & release. Need to find out why rpmdev-bumpspec didn't automatically add it...